lib/global-styles.php

```diff
@@ -586,6 +586,24 @@ function gutenberg_experimental_global_styles_flatten_styles_tree( $styles ) {
  * @return string Stylesheet.
  */
 function gutenberg_experimental_global_styles_get_stylesheet( $tree ) {
+
+	// Check if we can use cached.
+	$can_use_cached = (
+		( ! defined( 'WP_DEBUG' ) || ! WP_DEBUG ) &&
+		( ! defined( 'SCRIPT_DEBUG' ) || ! SCRIPT_DEBUG ) &&
+		( ! defined( 'REST_REQUEST' ) || ! REST_REQUEST ) &&
```

Comment:
@aristath @jorgefilipecosta would you happen to remember why this condition was added? Why a REST_REQUEST cannot be cached?

We aim to remove it at https://github.com/WordPress/gutenberg/pull/45679/files#r1025439309 but wanted to make sure I understand this properly and is safe to remove.